docs/core/install/linux-ubuntu.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Install .NET Core on Ubuntu - .NET Core
+description: Demonstrates the various ways to install .NET Core SDK and .NET Core Runtime on Ubuntu.
+author: thraka
+ms.author: adegeo
+ms.date: 06/01/2020
+---
+
+# Install .NET Core SDK and .NET Core Runtime on Ubuntu
+
+.NET Core is supported on Ubuntu. This article describes how to install .NET Core on Ubuntu. When an Ubuntu version falls out of support, .NET Core is no longer supported with that version. However, these instructions may help you to get .NET Core running on those versions, even though it isn't supported.
+
+## Supported distributions
+
+The following is a list of currently supported .NET Core releases and the versions of Ubuntu they're supported on. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Ubuntu reaches end-of-life](https://wiki.ubuntu.com/Releases).
+
+For the best compatibility, choose a long-term release (LTS) version of both Ubuntu and .NET Core.
+
+| .NET Core   | Ubuntu version                                           |
+|-------------|----------------------------------------------------------|
+| 2.1 (LTS)   | [16.04 (LTS) ✔️](#1604-), [18.04 (LTS) ✔️](#1804-), [19.10 ❌](#1910-), [20.04 (LTS) ✔️](#2004-) |
+| 3.1 (LTS)   | [16.04 (LTS) ✔️](#1604-), [18.04 (LTS) ✔️](#1804-), [19.10 ❌](#1910-), [20.04 (LTS) ✔️](#2004-) |
+| 5.0 Preview | [16.04 (LTS) ✔️](#1604-), [18.04 (LTS) ✔️](#1804-), [19.10 ❌](#1910-), [20.04 (LTS) ✔️](#2004-) |
+
+The following table is a list of .NET Core versions which are ❌ no longer supported. The downloads for these still remain. The Ubuntu version listed is the *last* LTS release they were supported on:
+
+| .NET Core | Ubuntu version (LTS) |
+|-----------|----------------------|
+| 3.0       | 16.04, 18.04         |
+| 2.2       | 16.04, 18.04         |
+| 2.0       | 14.04, 16.04, 18.04  |
```

Comment:
To be more concrete, I think the sections are just missing for these.

Reply:
I originally didn't include 14.04 because I didn't have that documentation actually created. I can't really verify it easily either. However, since it was never documented and is currently out of support I say we just get rid of that.

Reply:
That sounds good to me, since it's already not an exhaustive list.

Reply:
Now that I've been playing with it more and thinking about it. I wonder if we should reverse the tables and indicate which versions of .NET were supported and not the other way around.

---

The following table represents the support status of Ubuntu and .NET Core.

- A ✔️ indicates that the version of Ubuntu or .NET Core is still supported.
- A ❌ indicates that the version of Ubuntu or .NET Core is no longer supported.
- When both a version of Ubuntu and a version of .NET Core both have ✔️ that OS and .NET combination are supported.

| Ubuntu | .NET Core 2.x | .NET Core 3.x | .NET 5 Preview |
|-|-|-|-|
| ✔️ [20.04 (LTS)](#2004-)| ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |
| ✔️ [19.10](#1910-)| ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |
| ❌ [19.04](#1904-)| ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |
| ❌ [18.10](#1604-)| ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |
| ✔️ [18.04 (LTS)](#1804-)| ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |
| ❌ [17.10](#1804-)| ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |
| ❌ [17.04](#1804-)| ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |
| ❌ [16.10](#1804-)| ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |
| ✔️ [16.04 (LTS)](#1804-) | ❌ 2.2<br>✔️ 2.1<br>❌ 2.0 | ✔️ 3.1<br>❌ 3.0 | ✔️ 5.0 Preview |

Reply:
Showing distro versions as primary makes sense, but I'm not sure about the version columns... seems fairly noisy, for something that I don't think many people will be actively looking for. (I did like having this in a separate table, which seems a little harder to do with distro versions primary.)

Taking it from the top, I think the main entry point is this button:

> ![image](https://user-images.githubusercontent.com/12819531/83803122-a4864680-a671-11ea-858d-5954b54dfcb8.png)

or the tutorial which doesn't mention a version:

> ![image](https://user-images.githubusercontent.com/12819531/83803145-b1a33580-a671-11ea-9ff7-a8e1ac59f35d.png)

I expect that'll take me to https://review.docs.microsoft.com/en-us/dotnet/core/install/linux?branch=pr-en-us-18300. So maybe focusing on how it looks on this page is good? Trimming out the EOL .NET versions removes quite a bit of clutter (also updated availability a bit):

| Ubuntu | .NET Core 2.1 | .NET Core 3.1 | .NET 5 Preview |
|-|-|-|-|
| ✔️ [20.04 (LTS)](#2004-)| ✔️ 2.1 | ✔️ 3.1 | ✔️ 5.0 Preview |
| ✔️ [19.10](#1910-)| ✔️ 2.1 | ✔️ 3.1 | ✔️ 5.0 Preview |
| ❌ [19.04](#1904-)| ✔️ 2.1 | ✔️ 3.1 | ❌ 5.0 Preview |
| ❌ [18.10](#1604-)| ✔️ 2.1 | ❌ 3.1 | ❌ 5.0 Preview |
| ✔️ [18.04 (LTS)](#1804-)| ✔️ 2.1 | ✔️ 3.1 | ✔️ 5.0 Preview |
| ❌ [17.10](#1804-)| ✔️ 2.1 | ❌ 3.1 | ❌ 5.0 Preview |
| ❌ [17.04](#1804-)| ✔️ 2.1 | ❌ 3.1 | ❌ 5.0 Preview |
| ❌ [16.10](#1804-)| ❌ 2.1 | ❌ 3.1 | ❌ 5.0 Preview |
| ✔️ [16.04 (LTS)](#1804-) | ✔️ 2.1 | ✔️ 3.1 | ✔️ 5.0 Preview |

(I think it makes sense for anyone going after EOL .NET versions to have to click through to find more details--I think they'd reasonably be a more advanced user if they have a specific EOL version in mind.)

But maybe we shouldn't even have a table here? The instructions these would link to only describe how to install one version anyway (`dotnet-sdk-3.1`) so we aren't really saying how they could get 2.1 if they want it.